README.md

```diff
@@ -1,80 +1,98 @@
+those regions are accessed.
 
-Currently there is one, single-threaded application instance per device,
-however the application can employ any form of concurrency needed. In the
-future we plan to make libmuser multi-threaded. The application can be
-implemented in whatever way is convenient, e.g. as a Python script using
-bindings, on the cloud, etc. There's also experimental support for polling.
+Currently there is one, single-threaded, application instance per device,
+however the application can employ any form of concurrency needed. In the future
+we plan to make libvfio-user multi-threaded. The application can be implemented
+in whatever way is convenient, e.g. as a Python script using the bindings, on
+the cloud, etc. There's also experimental support for polling.
 
+The library (and the protocol) are actively under development, and should not be
+considered a stable API or interface. Work is underway to integrate the protocol
+with `qemu` (as a client) and [SPDK](https://spdk.io) (on the server side,
+implementing a virtual NVMe controller).
 
 Memory Mapping the Device
 -------------------------
 
 The device driver can allow parts of the virtual device to be memory mapped by
 the virtual machine (e.g. the PCI BARs). The business logic needs to implement
 the mmap callback and reply to the request passing the memory address whose
-backing pages are then used to satisfy the original mmap call. Currently
-reading and writing of the memory mapped memory by the client goes undetected
-by libmuser, the business logic needs to poll. In the future we plan to
-implement a mechanism in order to provide notifications to libmuser whenever a
-page is written to.
+backing pages are then used to satisfy the original mmap call. Currently reading
+and writing of the memory mapped memory by the client goes undetected by
+`libvfio-user`, the business logic needs to poll. In the future we plan to
+implement a mechanism in order to provide notifications to `libvfio-user`
+whenever a page is written to.
 
 
 Interrupts
 ----------
 
-Interrupts are implemented by passing the event file descriptor to libmuser
-and then notifying it about it. libmuser can then trigger interrupts simply by
-writing to it. This can be much more expensive compared to triggering interrupts
-from the kernel, however this performance penalty is perfectly acceptable when
-prototyping the functional aspect of a device driver.
+Interrupts are implemented by passing the event file descriptor to
+`libvfio-user` and then notifying it about it. `libvfio-user` can then trigger
+interrupts simply by writing to it. This can be much more expensive compared to
+triggering interrupts from the kernel, however this performance penalty is
+perfectly acceptable when prototyping the functional aspect of a device driver.
 
 
 Building muser
 ==============
 
+Build requirements are `cmake` (v2 or above) and `libjson-c-dev` /
+`libjson-c-devel`.
```

Comment:
Should we also add libcmocka-dev to requirements list? I remember my local build failed and was expecting libcmocka-dev installed.

Reply:
Yes, we should.

Reply:
done